README.md

```diff
@@ -251,6 +251,19 @@ If the `method` is `serverset` then we expect to find Finagle ServerSet
 (also used by [Aurora](https://github.com/apache/aurora/blob/master/docs/user-guide.md#service-discovery)) registrations with a `serviceEndpoint` and optionally one or more `additionalEndpoints`.
 The Synapse `name` will be automatically deduced from `shard` if present.
 
+##### Zookeeper Poll #####
+
+This watcher retrieves a list of servers and also service config data from zookeeper.
+Instead of setting Zookeeper watchers, it uses a long-polling method.
+
+It takes the following mandatory arguments:
+
+* `method`: zookeeper_poll
```

Comment:
why exposing "method"? Is it a flag to turn on/off polling? or you can switch btw poll and push?

Reply:
The `method` is used by each watcher to identify it (and the loader uses this value to find it). I.e. the original `method` for Zookeeper is just `zookeeper`

Reply:
IC, so the `zookeeper` means `zookeeper_push`, and it is already configured everywhere.

Reply:
Yup, that's correct